tracking/src/main/java/org/hps/recon/tracking/SensorSetup.java

```diff
@@ -16,6 +18,8 @@ public class SensorSetup extends RawTrackerHitSensorSetup {
 
     /// Name of the collection of fitted hits 
     private String fittedHitColName_ = ""; 
+    
+    private List<String> sensorNames_ = new ArrayList<String>();
```

Comment:
Change this to a HashSet. You don't need to initialize the array since it's you do so in the `setSkipSensors` method below.